src/ExportConverters.mss

```diff
@@ -653,18 +622,21 @@ function ConvertNoteStyle (style) {
             // and does not differentiate in the
             // head style, so we have to choose one
             // or the other.
-            noteStyle = 'cross';
+            noteStyle = 'x';
         }
         case (BlackAndWhiteDiamondNoteStyle)
         {
+            // There is not filled diamond in data.HEADSHAPE.list
```

Comment:
I think we should get rid of the non-standard names we currently use. In this case, we should use `'diamond'` if `noteobj.ParentNoteRest.Duration > 256` and otherwise [`'#xE0DB'`](https://smufl-browser.edirom.de/index?glyphname=noteheadDiamondBlack). In `GenerateNote()`, we'd then have to check for leading `'#x'` and set `@head.auth="smufl"` accordingly.